workflow/snakemake/Snakefile

```diff
@@ -0,0 +1,36 @@
+# Note that if you are working on the analysis development locally, i.e. outside
+# of the REANA platform, you can proceed as follows:
+#
+#   $ cd reana-demo-worldpopulation
+#   $ mkdir snakemake-local-run
+#   $ cd snakemake-local-run
+#   $ virtualenv ~/.virtualenvs/worldpopulation-snakemake
+#   $ source ~/.virtualenvs/worldpopulation-snakemake/bin/activate
+#   $ pip install snakemake
```

Comment:
As mentioned by @audrium in MM, we need to install some more deps here.

`pip install snakemake papermill ipykernel pandas matplotlib` should be enough.